sympy/matrices/matrices.py

```diff
@@ -1133,10 +1133,10 @@ def eigenvals(self, error_when_incomplete=True, **flags):
         return eigs
 
     def eigenvects(self, error_when_incomplete=True, **flags):
-        """Return list of triples (eigenval, multiplicity, basis).
+        """Return list of ``triples (eigenval, multiplicity, basis)``.
```

Comment:
I'm not sure the tics should be around this. This could also be phrased, "Return a tuple containing (in order) the eigenvalue, multiplicity and basis.